setup.py

```diff
@@ -15,7 +15,7 @@
     'google-apitools',
     'httplib2 >= 0.9.1',
     'oauth2client >= 1.4.6',
-    'protobuf == 3.0.0-alpha-1',
+    'protobuf >= 3.0.0a3',
```

Comment:
@tseaver Note that I am doing the same here as in #1157 (but I'd rather just rebase this after #1157 goes in)

This is because `protobuf >= 3.0.0a3` is required to make gRPC work.